.codeclimate.yml

```diff
@@ -0,0 +1,9 @@
+languages:
+  Ruby: false
```

Comment:
I don't think it's necessary to mark these as false, just the languages you want to cover should be marked as true.